packages/ERTP/src/amountMath.js

```diff
@@ -282,42 +282,98 @@ const AmountMath = {
    * @param {Brand=} brand
    * @returns {boolean}
    */
-  isEmpty: (amount, brand = undefined) => {
+  isEmpty: (amount, brand) => {
     assertRecord(amount, 'amount');
     const { brand: allegedBrand, value } = amount;
     assertRemotable(allegedBrand, 'brand');
     optionalBrandCheck(allegedBrand, brand);
     const h = assertValueGetHelpers(value);
     return h.doIsEmpty(h.doCoerce(value));
   },
+  /**
+   * Returns true if the leftAmount is greater than (and not equal to) the
+   * rightAmount.
+   *
+   * For non-scalars, "greater than or equal to" depends on the kind of amount,
+   * as defined by the MathHelpers. For example, whether rectangle A is greater
+   * than rectangle B depends on whether rectangle A includes rectangle B as
+   * defined by the logic in MathHelpers.
+   *
+   * @template {AssetKind} [K=AssetKind]
+   * @param {Amount<K>} leftAmount
+   * @param {Amount<K>} rightAmount
+   * @param {Brand<K>} [brand]
+   * @returns {boolean}
+   */
+  isGT: (leftAmount, rightAmount, brand) => {
+    return !AmountMath.isGTE(rightAmount, leftAmount, brand);
+  },
```

Comment:
This definition is wrong for partial orders. Instead
```suggestion
  isGT: (leftAmount, rightAmount, brand = undefined) => {
    return AmountMath.isGTE(leftAmount, rightAmount, brand) && !AmountMath.isEqual(leftAmount, rightAmount);
  },
```
which directly mirrors your prose definition above.